pt-BR/action_controller_overview.md

```diff
@@ -760,55 +760,55 @@ O `form_authenticity_token` gera um *token* de autenticação válido. Isso é 
 
 O [Guia de segurança](security.html) possui mais informações sobre isso e muitos outros problemas relacionados a segurança que você deve estar ciente quando desenvolve uma aplicação *web*.
 
-The Request and Response Objects
+Os Objetos de Requisição e Resposta
 --------------------------------
 
-In every controller there are two accessor methods pointing to the request and the response objects associated with the request cycle that is currently in execution. The `request` method contains an instance of `ActionDispatch::Request` and the `response` method returns a response object representing what is going to be sent back to the client.
+Em todo *controller* existem dois métodos acessores apontando para os objetos de requisição e de resposta associados com o ciclo de requisição que estiver em execução no momento. O método `request` contém uma instância de `ActionDispatch::Request` e o método `response` retorna um objeto de resposta representando o que será enviado de volta ao cliente.
 
-### The `request` Object
+### O Objeto `request`
 
-The request object contains a lot of useful information about the request coming in from the client. To get a full list of the available methods, refer to the [Rails API documentation](https://api.rubyonrails.org/classes/ActionDispatch/Request.html) and [Rack Documentation](https://www.rubydoc.info/github/rack/rack/Rack/Request). Among the properties that you can access on this object are:
+O objeto de requisição contém muitas informações úteis sobre a requisição proveniente do cliente. Para obter uma lista completa dos métodos disponíveis verifique a [documentação da API do Rails](https://api.rubyonrails.org/classes/ActionDispatch/Request.html) e a [documentação do Rack](https://www.rubydoc.info/github/rack/rack/Rack/Request). Entre as propriedades que você pode acessar estão:
 
-| Property of `request`                     | Purpose                                                                          |
-| ----------------------------------------- | -------------------------------------------------------------------------------- |
-| host                                      | The hostname used for this request.                                              |
-| domain(n=2)                               | The hostname's first `n` segments, starting from the right (the TLD).            |
```

Comment:
Não consegui identificar o que significa esse techo no final da descrição da propriedade `domain` - esse "the TLD". :thinking:

Reply:
`TLD` significa `Top Level Domain`. É basicamente a parte final da URL.

Por exemplo, na URL `https://www.pudim.com.br`, o `Top Level Domain` é a parte `.com.br`.
Podemos traduzir como `domínio de primeiro nível`, o que você acha? 😄

Reply:
Olá. = )  
Muito obrigado pelas informações. o/  
  
Acho que fica bom sim `domínio de primeiro nível`, vou ajustar.